packages/wallet-sdk/src/vendor-js/eth-eip712-util/index.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable */
+//prettier-ignore
+
```

Comment:
why is this needed now but wasn't before if there are no other changes in this file? 🤔

Reply:
I was logging in this file to trace down the issue and lint was trying to update everything. To prevent that moving forward it made sense to ignore this file as well